File: netlify/fix.js
const fs = require('fs');
const content = fs.readFileSync('netlify/functions/confirm-ticket-payment.js', 'utf8');
const lines = content.split('\n');
const fixed = lines.map(line => {
  if (line.includes('hostname:')) {
    return "      hostname: 'firestore.googleapis.com',";
  }
  return line;
}).join('\n');
fs.writeFileSync('netlify/functions/confirm-ticket-payment.js', fixed);
console.log('Done!');

File: netlify/fix.py
with open('netlify/functions/confirm-ticket-payment.js', 'r') as f:
    content = f.read()

# Find and replace the bad hostname
import re
content = re.sub(r"hostname: '[^']*'", "hostname: 'firestore.googleapis.com'", content)

with open('netlify/functions/confirm-ticket-payment.js', 'w') as f:
    f.write(content)

print('Done! hostname is now:')
import subprocess
subprocess.run(['grep', 'hostname', 'netlify/functions/confirm-ticket-payment.js'])